engine/render/src/render/render_script.cpp

```diff
@@ -2645,6 +2717,10 @@ namespace dmRender
         lua_setfield(L, -2, "BUFFER_"#name);
 
         REGISTER_BUFFER_CONSTANT(COLOR_BIT);
+        REGISTER_BUFFER_CONSTANT(COLOR0_BIT);
```

Comment:
I think the dmGraphics::xxCOLOR_BIT should be removed, and instead be set here to COLOR0_BIT.